components/tools/OmeroPy/src/omero/cli.py

```diff
@@ -802,6 +802,12 @@ def _ask_for_password(self, reason="", root_pass=None, strict=True):
             break
         return root_pass
 
+    def _add_wait(self, parser, default=-1):
```

Comment:
Then that's likely something in the logic somewhere (i.e. a bug) since `-1` should mean "wait forever"